File: java/ContactsExample.java
public class ContactsExample {
    public static void main(String[] args) {
        try {
            MailproClient client = new MailproClient("YOUR_API_KEY");

            String listId = "12345";
            String email = "newcontact@example.com";

            String response = client.addContact(listId, email);
            System.out.println("Add Contact Response: " + response);

        } catch (Exception e) {
            e.printStackTrace();
        }
    }
}

File: php/ContactsExample.php
<?php
require_once "MailproClient.php";

$client = new MailproClient("YOUR_API_KEY");

try {
    $response = $client->addContact(
        "LIST_ID_HERE",
        "newcontact@example.com",
        "John",
        "Doe"
    );
    print_r($response);
} catch (Exception $e) {
    echo "Error: " . $e->getMessage();
}
?>
